src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -68,4 +70,21 @@ void auditRateOf(String key) {
             LOGGER.error("Unexpected error creating a Rate Limiter object in cache", e);
         }
     }
+
+    private int getRateForMethod(String method){
+        if(METHOD_POST.equals(method)){
+            return rateForPost;
+        }
+        return rate;
+    }
+
+    /**
+     * Returns rate to use local cache. It is the rate divided by no. of publicapi nodes
+     * 
+     * @param method GET, POST ...
+     * @return
+     */
```

Comment:
no need for javadoc here as the method itself is self documenting

Reply:
Remove javadoc